test/parallel/test-assert.js

```diff
@@ -839,7 +832,7 @@ common.expectsError(
     {
       type: assert.AssertionError,
       code: 'ERR_ASSERTION',
-      message: 'Got unwanted exception.\nundefined'
+      message: 'Got unwanted exception.\nActual message: undefined'
```

Comment:
This still uses `common.expectsError()`, is this wanted?

Reply:
Exactly. This call seems similar to https://github.com/nodejs/node/pull/19884/files#diff-81c8c31c5728ec9ed47700e55d9ceebfL127.

Can't this be changed to `assert.throws(...)` as well?